options/posix/include/sys/ipc.h

```diff
@@ -14,4 +18,16 @@
 
 typedef int key_t;
 
+struct ipc_perm {
+	key_t __ipc_perm_key;
+	uid_t uid;
+	gid_t gid;
+	uid_t cuid;
+	gid_t cgid;
+	mode_t mode;
+	int __ipc_perm_seq;
+	long __pad1;
+	long __pad2;
```

Comment:
I think we can drop the padding if Linux doesn't use it. Are the double underscore fields used in Linux?